tests/e2e/support.py

```diff
@@ -45,3 +59,27 @@ def run_oc_command(args):
     except subprocess.CalledProcessError as e:
         print(f"Error executing 'oc {' '.join(args)}': {e}")
         return None
+
+
+def create_local_queue(self, cluster_queue):
+    json = {
+        "apiVersion": "kueue.x-k8s.io/v1beta1",
+        "kind": "LocalQueue",
+        "metadata": {
+            "annotations": {"kueue.x-k8s.io/default-queue": "true"},
+            "namespace": self.namespace,
+            "name": "local-queue",
+        },
+        "spec": {"clusterQueue": cluster_queue},
+    }
+    try:
+        self.custom_api.create_namespaced_custom_object(
+            group="kueue.x-k8s.io",
+            namespace=self.namespace,
+            plural="localqueues",
+            version="v1beta1",
+            body=json,
+        )
+    except Exception as e:
+        print(f"An unexpected error occurred while creating 'local-queue'. Error: ", e)
+        delete_namespace(self)
```

Comment:
Same point as above except call `delete_namespace(self)` before `_kube_api_error_handling` in the exception